static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
I'd be interested to hear what the specific issue was with the module name not being normalized? Is the normalization configuration not present in production?